index.html

```diff
@@ -19,6 +19,11 @@
 
 	<h1 class="d2l-heading-2">Demos</h1>
 
+	<ul>
+		<li><b>Build version:</b> __buildVersion__</li>
+		<li><b>Build date:</b> __buildDate__</li>
+	</ul>
```

Comment:
Another option here that should let you use an off-the-shelf plugin would be to build a small web component do to this. That way, the replacement would be a in a JS file. It could also probably be a bit smart and if the replacement didn't occur -- like in dev -- it could just plug in some default values. That would remove the need for the duplicate plugin and the `web-dev-sever.config`. 🤷

Reply:
Yeah that works, or could even just be defined inline in this file inside a `<script type="module">`.

Reply:
I considered that, but inline means it's not in a JS file, which means `@rollup/plugin-replace` won't replace it 😂.

Reply:
Hahaha, right.